lib/dgram.js

```diff
@@ -241,21 +243,14 @@ Socket.prototype.sendto = function(buffer,
   this.send(buffer, offset, length, port, address, callback);
 };
 
-
-Socket.prototype.send = function(buffer,
-                                 offset,
-                                 length,
-                                 port,
-                                 address,
-                                 callback) {
-  var self = this;
-
+function sliceBuffer(buffer, offset, length) {
```

Comment:
The offset checks here are no longer necessary since we've moved to typed arrays.